src/plugins/recordTypes/uoc/fields.js

```diff
@@ -0,0 +1,310 @@
+                type: TermPickerInput,
+                props: {
+                  source: 'uocmethods',
+                },
+              },
+            },
+          },
+        },
+        title: {
+          [config]: {
+            cloneable: false,
+            messages: defineMessages({
+              name: {
+                id: 'field.uoc_common.title.name',
+                defaultMessage: 'Title',
+              },
+            }),
+            view: {
+              type: TextInput,
+            },
+          },
+        },
+        authorizedBy: {
+          [config]: {
+            messages: defineMessages({
+              fullName: {
+                id: 'field.uoc_common.authorizedBy.fullName',
+                defaultMessage: 'Authorized by',
+              },
+              name: {
+                id: 'field.uoc_common.authorizedBy.name',
+                defaultMessage: 'Authorized by',
+              },
+            }),
+            view: {
+              type: AutocompleteInput,
+              props: {
+                source: 'person/local,person/shared',
+              },
+            },
+          },
+        },
+        authorizationDate: {
+          [config]: {
+            dataType: DATA_TYPE_DATE,
+            messages: defineMessages({
+              fullName: {
+                id: 'field.uoc_common.authorizationDate.fullName',
+                defaultMessage: 'Use of Collections authorization date',
+              },
+              name: {
+                id: 'field.uoc_common.authorizationDate.name',
+                defaultMessage: 'Authorization date',
+              },
+            }),
+            view: {
+              type: DateInput,
+            },
+          },
+        },
+        authorizationNote: {
+          [config]: {
+            messages: defineMessages({
+              name: {
+                id: 'field.uoc_common.authorizationNote.name',
+                defaultMessage: 'Authorization Note',
+              },
+            }),
+            view: {
+              type: TextInput,
+            },
+          },
+        },
+        startSingleDate: {
+          [config]: {
+            dataType: DATA_TYPE_DATE,
+            messages: defineMessages({
+              fullName: {
+                id: 'field.uoc_common.startSingleDate.fullName',
+                defaultMessage: 'Use of Collections start/single date',
+              },
+              name: {
+                id: 'field.uoc_common.startSingleDate.name',
+                defaultMessage: 'Start/single date',
+              },
+            }),
+            view: {
+              type: DateInput,
+            },
+          },
+        },
+        endDate: {
+          [config]: {
+            dataType: DATA_TYPE_DATE,
+            messages: defineMessages({
+              fullName: {
+                id: 'field.uoc_common.endDate.fullName',
+                defaultMessage: 'Use of Collections end date',
+              },
```

Comment:
Ditto above. We know we're on use of collections record, so I don't think this message is needed.